src/main/java/org/caffa/rpc/CaffaObjectArrayField.java

```diff
@@ -24,21 +24,27 @@ public CaffaObjectArrayField(CaffaObject owner, String keyword) {
 
         ArrayList<CaffaObject> objects = new ArrayList<>();
 
-        for (RpcObject object : objectList.getObjectsList())
-        {
+        for (RpcObject object : objectList.getObjectsList()) {
             String jsonString = object.getJson();
             System.out.println("Got child JSON: " + jsonString);
```

Comment:
Log instead of System.out.println?

Reply:
Agree. I will go over and remove all printouts in a separate commit